ui/webui/test/check-storage

```diff
@@ -20,30 +20,33 @@ import sys
 
 # import Cockpit's machinery for test VMs and its browser test API
 TEST_DIR = os.path.dirname(__file__)
+sys.path.append(TEST_DIR)
 sys.path.append(os.path.join(TEST_DIR, "common"))
 sys.path.append(os.path.join(os.path.dirname(TEST_DIR), "bots/machine"))
 
+from common_helpers import CommonHelpers
+from storage_helpers import StorageHelpers
 from testlib import MachineCase, nondestructive, test_main  # pylint: disable=import-error
 from machine_install import VirtInstallMachine
 
 @nondestructive
-class TestStorage(MachineCase):
+class TestStorage(MachineCase, CommonHelpers, StorageHelpers):
     MachineCase.machine_class = VirtInstallMachine
 
     def testLocalStandardDisks(self):
         b = self.browser
 
-        b.open("/cockpit/@localhost/anaconda-webui/index.html")
+        self.open()
```

Comment:
Will this work?. I mean `CommonHelpers` has the open pointing to ``"/cockpit/@localhost/anaconda-webui/index.html#/installation-language"``.

Reply:
This is pointing to initial page anyway, to yes, it will work.